supertokens_python/recipe/emailverification/interfaces.py

```diff
@@ -49,26 +50,18 @@ def __init__(self):
         self.is_email_already_verified = True
 
 
-class VerifyEmailUsingTokenResult(ABC):
-    def __init__(
-            self, status: Literal['OK', 'EMAIL_VERIFICATION_INVALID_TOKEN_ERROR'], user: Union[User, None]):
-        self.status = status
-        self.is_ok = False
-        self.is_email_verification_invalid_token_error = False
-        self.user = user
-
-
-class VerifyEmailUsingTokenOkResult(VerifyEmailUsingTokenResult):
+class VerifyEmailUsingTokenOkResult:
     def __init__(self, user: User):
-        super().__init__('OK', user)
+        self.status = 'OK'
+        self.user = user
         self.is_ok = True
         self.is_email_verification_invalid_token_error = False
 
 
-class VerifyEmailUsingTokenInvalidTokenErrorResult(
-        VerifyEmailUsingTokenResult):
+class VerifyEmailUsingTokenInvalidTokenErrorResult:
     def __init__(self):
-        super().__init__('EMAIL_VERIFICATION_INVALID_TOKEN_ERROR', None)
+        self.status = 'EMAIL_VERIFICATION_INVALID_TOKEN_ERROR'
+        self.user = None
```

Comment:
You don't need this type in this class. Not sure why this is assigned to None

Reply:
done.